src/PokemonGeneral.java

```diff
@@ -0,0 +1,58 @@
+abstract class PokemonGeneral {
+    private String name;
+    private String type;
+    private int hp;
+    private int level;
+
+    public PokemonGeneral(String name, int hp, int level) {
+        this.name = name;
+        this.hp = hp;
+        this.level = level;
+    }
+
+    abstract void sound();
+
+    public void tackle(String name) {
+
+        System.out.println(name + " doet je nu tackelen");
+    }
+
+    public void reward(int hp) {
+        this.hp = hp + 10;
```

Comment:
Waarom `+10`? 
Als ik nu een reward van 10 wil geven zeg ik `pikachu.reward(10)` , maar dan krijgt pikachu er werkelijk 20. 

Misschien beter de meegegeven hp bij de originele hp op kunt tellen door: 
`this.hp = this.hp + hp`

Let hierbij op dat `hp` en `this.hp` twee verschillende variabelen zijn.

Reply:
De print statement hieronder suggereert dat je eigenlijk helemaal geen parameter nodig hebt, maar gewoon `+10` wilt doen. In dat geval wordt het:
`this.hp += 10` of `this.hp = this.hp + 10`
Je hebt dan geen parameter nodig